agent_api_rest/src/holder/mod.rs

```diff
@@ -11,13 +11,19 @@ use crate::API_VERSION;
 use agent_holder::state::HolderState;
 use axum::routing::get;
 use axum::{routing::post, Router};
+use holder::presentations::{get_presentations, post_presentations, presentation_signed::presentation_signed};
 
 pub fn router(holder_state: HolderState) -> Router {
     Router::new()
         .nest(
             API_VERSION,
             Router::new()
                 .route("/holder/credentials", get(credentials))
+                .route("/holder/presentations", get(get_presentations).post(post_presentations))
+                .route(
+                    "/holder/presentations/:presentation_id/signed",
```

Comment:
Because `GET /holder/presentations/:id` would return the Presentation aggregate object (which has a property called `signed` which is the actual signed presentation JWT)

Reply:
The endpoint will be added in the [next PR](https://github.com/impierce/ssi-agent/pull/130)